docs/apis/http/tus_upload.md

```diff
@@ -191,3 +220,31 @@ curl -ks -XPOST https://ocis.test/remote.php/dav/spaces/{space-id} \
+| spaceName              | Personal                                                                                                                                             | Sent by the web UI                        |
+| driveAlias             | personal/admin                                                                                                                                       | Sent by the web UI                        |
+| driveType              | personal                                                                                                                                             | Sent by the web UI                        |
+| currentFolder          | /                                                                                                                                                    | Sent by the web UI                        |
+| currentFolderId        | ODc0OGNkZGYtNjZiNy00Yjg1LTkxYTctZTZkMDhkOGUxNjM5JGE5Nzc4ZDYzLTIxZTctNGQ5Mi05YjQ3LTFiODExNDRiOTk5MyFhOTc3OGQ2My0yMWU3LTRkOTItOWI0Ny0xYjgxMTQ0Yjk5OTM= | Sent by the web UI                        |
+| uppyId                 | uppy-example/pdf-1e-application/pdf-238300                                                                                                           | Sent by the web UI                        |
+| relativeFolder         |                                                                                                                                                      | Key sent by the web UI, but without value |
+| tusEndpoint            | https://ocis.ocis-traefik.latest.owncloud.works/remote.php/dav/spaces/8748cddf-66b7-4b85-91a7-e6d08d8e1639$a9778d63-21e7-4d92-9b47-1b81144b9993      | Sent by the web UI                        |
+| uploadId               | 71d5f878-a96c-4d7b-9627-658d782c93d7                                                                                                                 | Sent by the web UI                        |
+| topLevelFolderId       | undefined                                                                                                                                            | Sent by the web UI                        |
+| routeName              | files-spaces-generic                                                                                                                                 | Sent by the web UI                        |
+| routeDriveAliasAndItem | cGVyc29uYWwvYWRtaW4=                                                                                                                                 | Sent by the web UI                        |
+| routeShareId           |                                                                                                                                                      | Key sent by the web UI, but without value |
```

Comment:
It only has a value when you upload something into a folder that has been shared with you. Web (currently) needs this ID to correctly resolve into the folder after the upload is finished.